Watchman.Discord/Areas/Protection/Controllers/MuteUserController.cs

```diff
@@ -15,18 +15,20 @@ public class MuteUserController : IController
         private readonly MessagesServiceFactory _messagesServiceFactory;
         private readonly MuteService _muteService;
         private readonly UsersService _usersService;
+        private readonly DirectMessagesService _directMessagesService;
 
-        public MuteUserController(MessagesServiceFactory messagesServiceFactory, MuteService muteService, UsersService usersService)
+        public MuteUserController(MessagesServiceFactory messagesServiceFactory, MuteService muteService, UsersService usersService, DirectMessagesService directMessagesService)
         {
             this._messagesServiceFactory = messagesServiceFactory;
             this._muteService = muteService;
             this._usersService = usersService;
+            this._directMessagesService = directMessagesService;
         }
 
         //[IgnoreForHelp] todo:
         [DiscordCommand("mute")]
         [AdminCommand]
-        public void MuteUser(DiscordRequest request, Contexts contexts)
+        public async void MuteUser(DiscordRequest request, Contexts contexts)
```

Comment:
Tak teraz patrzę, że w StatisticsController mamy async Task i działa, więc w sumie możemy we wszystkich inny kontrolerach pozamieniać async void na async Task
```suggestion
        public async Task MuteUser(DiscordRequest request, Contexts contexts)
```